components/Common/CodeBox/CodeTabs/index.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { FC } from 'react';
+
+import { CodeBox } from '@/components/Common/CodeBox';
+import { CodeTabs } from '@/components/Common/CodeBox/CodeTabs';
+
+type Story = StoryObj<typeof CodeTabs>;
+type Meta = MetaObj<typeof CodeTabs>;
+
+const TabsContent: FC = () => (
+  <>
+    <TabsPrimitive.Content key="mjs" value="mjs">
+      <CodeBox language="JavaScript (MJS)">
+        <code>
+          {`import * as http from 'http';
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});`}
+        </code>
+      </CodeBox>
+    </TabsPrimitive.Content>
+    <TabsPrimitive.Content key="cjs" value="cjs">
+      <CodeBox language="JavaScript (CJS)">
+        <code>
+          {`const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});`}
+        </code>
+      </CodeBox>
+    </TabsPrimitive.Content>
+  </>
+);
+
+export const Default: Story = {};
+
+export const WithMoreOptions: Story = {
+  args: {
+    linkUrl: 'https://github.com/nodejs/nodejs.org',
+    linkText: 'More options',
+  },
+};
+
+export default {
+  component: CodeTabs,
+  args: {
+    children: <TabsContent />,
+    defaultValue: 'mjs',
+    tabs: [
+      {
+        key: 'mjs',
+        label: 'MJS',
+      },
+      {
+        key: 'cjs',
+        label: 'CJS',
+      },
+    ],
+  },
+  parameters: {
```

Comment:
What are these controls?

Reply:
The available props to control the story. In the case of CodeBox and CodeTabs there is nothing controllable, so by not including any prop it leaves us with an empty control tab.

![image](https://github.com/nodejs/nodejs.org/assets/23018799/e14c4731-4f2f-4b96-8c70-5dc4bc6de908)

Reply:
I don't think you even need to add this. It should still work normally (just by removing it) imo.